src/theme/theme.ts

```diff
@@ -78,6 +79,7 @@ declare module "@mui/material/styles" {
       grey500?: string;
       grey600?: string;
       grey700?: string;
+      lightGreen?: string;
```

Comment:
I added this as it was a new color seen in the DSP Pricing Plan Figma design but I need to have the conversation with Design to double check consistency across the design and nomenclature.